app/src/main/java/org/mozilla/focus/home/HomeFragment.kt

```diff
@@ -261,8 +261,8 @@ private fun onBindCustomHomeTile(uiLifecycleCancelJob: Job, holder: TileViewHold
         val validUri = item.url.toJavaURI()
 
         val screenshotDeferred = async {
-            val screenshot = HomeTileScreenshotStore.read(itemView.context, item.id)
-            screenshot ?: HomeTilePlaceholderGenerator.generate(itemView.context, item.url)
+            val screenshot = HomeTileScreenshotStore.read(itemView.context, item.id)?.withRoundedCorners(itemView.resources.getDimension(R.dimen.home_tile_corner_radius))
```

Comment:
nit: we shouldn't use such long line lengths because it makes code hard to read. That being said, this will get fixed when we set up detektCheck.